src/main/resources/i18n/strings.properties

```
@@ -168,10 +168,15 @@ lock.fail.message=Locking vault failed
 lock.fail.description=Vault "%s" could not be locked. Ensure unsaved work is saved elsewhere and important Read/Write operations are finished. In order to close the vault, kill the Cryptomator process.
 
 # Migration
-migration.title=Upgrade Vault
+migration.title=Migrate Vault
```

```suggestion
migration.title=Upgrade Vault
```

Let's keep it at "Upgrade", even though the technical term suggests "migration". I'm sure we had several discussions about this already. 😅

---

No problem! I switched back all of the terms in the latest commit.